Automattic/wc-calypso-bridge
# Add masterbar to setup wizard

Fixes #70 

This PR overrides the default `WC_Admin_Setup_Wizard` class so that we can add in the masterbar and other template changes.

This PR is branched from https://github.com/Automattic/wc-calypso-bridge/pull/72 which includes the initial setup fxns.

### Screenshots

<img width="1653" alt="screen shot 2018-10-22 at 5 08 50 pm" src="https://user-images.githubusercontent.com/10561050/47322349-35371c80-d61d-11e8-9a7f-80faefd24d6c.png">

### Testing

1.  Visit `/wp-admin/admin.php?page=wc-setup&calypsoify=1`
2.  Verify that admin bar (masterbar) is shown and calypsoified.
